urlparse4/__init__.py

```diff
@@ -10,4 +10,4 @@
 _original_urljoin = urljoin
 _original_urlparse = urlparse
 
-from cgurl import urlsplit, urljoin, urlparse, canonicalize_url
+from cgurl import urlsplit, urljoin, urlparse, canonicalize_url, parse_url
```

Comment:
``parse_url`` is not a standard function, do we need to expose it here?

Reply:
> Do you think it is better to make modification for this function in w3lib instead?

no, I just meant not exposing it here in this file, as public API, and instead importing it directly where it's needed instead (unless we want users of urlparse4 to call it)

Reply:
So I should just import it only in the test functions that need it right?

Reply:
Oh I get what you mean!